engine-tests/src/test/java/org/terasology/rendering/dag/RenderTaskListGeneratorTest.java

```diff
@@ -36,13 +36,13 @@ public void testSimpleReducePersistingStateChanges() {
         List<RenderPipelineTask> taskList = renderTaskListGenerator.generateFrom(orderedNodes);
 
         assertEquals("----- AlphaNode",        taskList.get(0).toString().trim()); // Strictly speaking we don't need
-        assertEquals("SetNameTask: foo",       taskList.get(1).toString().trim()); // trimming MarkerTask.toString(),
-        assertEquals("AlphaNode: process()",   taskList.get(2).toString().trim()); // resulting in "----- <NodeName>"
+        assertEquals("SetName: foo",       taskList.get(1).toString().trim()); // trimming MarkerTask.toString(),
```

Comment:
Once you are happy with the changes you did to the actual code, please update the test code in its own commit.